certified-connectors/UiPath Orchestrator/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3420 @@
+{
+  "swagger": "2.0",
+  "info":
+  {
+    "title": "UiPath Orchestrator",
+    "description": "Run automations in UiPath Cloud Orchestrator.",
+    "version": "1.0.0",
+    "contact":
+    {
+      "name": "Integrations Team",
+      "email": "integrations-apps@uipath.com",
+      "url": "https://www.uipath.com/support"
+    }
+  },
+  "host": "cloud.uipath.com",
+  "basePath": "/orchestrator_/",
+  "schemes":
+  [
+    "https"
+  ],
+  "x-ms-connector-metadata":
+  [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.uipath.com"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.uipath.com/legal/privacy-policy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "AI;Business Intelligence"
+    }
+  ],
+  "securityDefinitions":
```

Comment:
Please remove this section since this is also available in apiProperties.

Reply:
@mamurshe When this is removed the connector deploys but with no security configuration, "No Authentication".

Reply:
That happens when you directly upload the swagger. Remove this and use PACONN Cli to update.